Add RU translation: Learn → Getting started (index) (#28916)
mdn/translated-content: files/ru/learn_web_development/getting_started/index.md

## Предварительные требования

Предварительных знаний для начала этого курса не требуется. Также обратите внимание, что мы не считаем изучение этих тем обязательным перед переходом к [Основным модулям](/ru/docs/Learn_web_development/Core), но уверены, что вам будет проще, если вы сначала уделите немного времени этим материалам.

## Модули

- [Настройка окружения](/ru/docs/Learn_web_development/Getting_started/Environment_setup)
  - : В модуле _«Настройка окружения»_ мы покажем, какие инструменты нужны для простой веб-разработки, как правильно их установить, а также поможем разобраться в важных аспектах вашей среды — таких как файловые системы и командная строка.
- [Ваш первый веб-сайт](/ru/docs/Learn_web_development/Getting_started/Your_first_website)
  - : Этот модуль знакомит с практической стороной веб-разработки. Вы соберёте ресурсы и напишете код, чтобы создать и опубликовать простую веб-страницу.
- [Веб-стандарты](/ru/docs/Learn_web_development/Getting_started/Web_standards)
  - : В этом модуле рассматриваются основы того, как работает веб на высоком уровне: модель взаимодействия, ключевые технологии, как они создаются, и как браузер рендерит и отображает сайты пользователю.

> - [Soft skills](/ru/docs/Learn_web_development/Getting_started/Soft_skills)
>   - : Здесь вы найдёте рекомендации по развитию «мягких» навыков, которые стоит прокачивать во время обучения веб-разработке и при входе в индустрию. Они помогают сформировать правильные установки для обучения, ресёрча и коллаборации, а также повышают шансы на трудоустройство.

Reviewer: ```suggestion
- [Нетехнические навыки](/ru/docs/Learn_web_development/Getting_started/Soft_skills)
  - : Модуль содержит рекомендации по развитию «гибких» навыков, которые можно развивать в процессе обучения веб-разработке и которые являются полезными качествами при выходе на рынок труда. Они окажут огромную помощь в формировании правильного подхода к обучению, исследованиям и сотрудничеству, а также повысят ваши шансы получить работу.
```

> ## См. также

Reviewer: ```suggestion
## Смотрите также
```

> - [Путь фронтенд-разработчика](https://scrimba.com/the-frontend-developer-career-path-c0j?via=mdn) <sup>[_Партнёр по обучению MDN_](/ru/docs/MDN/Writing_guidelines/Learning_content#partner_links_and_embeds)</sup>
>   - : [Scrimba](https://scrimba.com?via=mdn) предлагает курс _Frontend Developer Career Path_, который научит всему, что нужно для уверенной работы фронтенд-разработчиком: интерактивные уроки и задания, опытные преподаватели и поддерживающее сообщество. Пройдите путь от нуля до первой работы во фронтенде! Многие компоненты курса доступны бесплатно в виде отдельных модулей.

Reviewer: ```suggestion
- [The Frontend Developer Career Path](https://scrimba.com/the-frontend-developer-career-path-c0j?via=mdn) <sup>[_Партнёр по обучению MDN_](/ru/docs/MDN/Writing_guidelines/Learning_content#partner_links_and_embeds)</sup> (англ.)
  - : Курс _Frontend Developer Career Path_ от [Scrimba](https://scrimba.com?via=mdn) научит всему, что необходимо знать фронтенд-разработчику. Это интерактивные уроки и задания, опытные преподаватели и дружелюбное сообщество. Пройдите путь с самого начала до первой работы во фронтенде! Многие компоненты курса доступны бесплатно в виде отдельных модулей.
```
